tests/dem/distribution_lognormal.cc

```diff
@@ -0,0 +1,137 @@
+// SPDX-FileCopyrightText: Copyright (c) 2023-2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+/**
+ * @brief Inserting particles following a lognormal distribution.
+ */
```

Comment:
Can you explain a bit more what the test does.
You insert particles following a log normal distribution and you verify that it is correct according to?